### ISSUE TITLE
Claim process (Rewards panel)

### ISSUE THREAD

**Issue by ryanml** [brave/browser-android-tabs#1216]
Original issue: https://github.com/brave/browser-android-tabs/issues/1148

Per @jamesmudgett the "claim" text should be replaced with a generic loader spinner

**Comment by LaurenWags** [comment]
This one seems pretty important. When I followed the steps below I completely screwed up my Rewards instance and couldn't use Rewards until I reinstalled:
1. Join Rewards
2. From the panel (accessed via BAT logo), tap on Claim button to claim a grant. 
3. No UI change, so tap a couple more times. 
4. Navigate to chrome://rewards, page is blank.
5. Tap on BAT logo, you didn't get the grant -> balance still displays zero.
6. Opening in a new tab, restarting Brave, restarting your phone - nothing works. Your Rewards instance is completely borked.

Using Asus Zenfone with Android 6.0 and 1.0.85 (BR_5)
